frontend/packages/ui/src/components/Toast/Toast.module.css

```diff
@@ -64,44 +68,12 @@
   color: var(--color-gray-1000);
 }
 
-.wrapper.info .title {
-  color: var(--info-foreground);
-}
-
-.wrapper.error .title {
-  color: var(--error-foreground);
-}
-
-.wrapper.success .title {
-  color: var(--success-foreground);
-}
-
-.wrapper.warning .title {
-  color: var(--warning-foreground);
-}
```

Comment:
`.title`s and `.description`s in the same wrappers share the same colors.
I moved this `color` declaration to the to the wrappers to simplify.